docs/FEATURES.md

```
@@ -0,0 +1,435 @@
+| `ntd daemon stop` | 停止服务 |
+| `ntd daemon restart` | 重启服务 |
+| `ntd daemon status` | 查看运行状态 (支持 `--verbose` 查看最近日志) |
+| `--force` | 强制重新安装 / 强制系统级操作 |
+
+---
+
+## 十七、CLI 命令行
+
+> 完整命令参考见 [ntd-cli.md](./ntd-cli.md)。下表为高层概览。
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)